resolvers/relay.ts

```diff
@@ -0,0 +1,66 @@
+export type RelayInformation = {
+    name?: string;
+    description?: string;
+    pubkey?: string;
+    contact?: string;
+    supported_nips?: number[];
+    software?: string;
+    version?: string;
+    icon?: string;
+};
+
+export const Information = async (kv: Deno.Kv) => {
+        const res = await kv.get<RelayInformation>(["relay_information"]);
+        if (res.value == null) return null;
+        return res.value;
+}
+
+export class RelayInformationStore {
+    not_modifiable_information: RelayInformation = {
+        software: "https://github.com/BlowaterNostr/relayed",
+        supported_nips: [1, 2, 11],
+        version: "RC5",
+    };
+    default_information: RelayInformation
+
+    constructor(
+        private kv: Deno.Kv,
+        default_information?: RelayInformation,
+    ) {
+        this.default_information = default_information?default_information:{};
+    }
+
+    resolveRelayInformation = async (): Promise<RelayInformation> => {
+        const get_relay_information = await Information(this.kv);
```

Comment:
Why define a separate `Information` function when this class already has a `resolveRelayInformation` method?

Let's simplify and remove `Information` and inline its content to this method directly

Reply:
ok